diff --git a/node_modules/expo-react-native-wechat-v2/ios/WeChat.h b/node_modules/expo-react-native-wechat-v2/ios/WeChat.h
index 92dffe2..85148f5 100644
--- a/node_modules/expo-react-native-wechat-v2/ios/WeChat.h
+++ b/node_modules/expo-react-native-wechat-v2/ios/WeChat.h
@@ -3,7 +3,7 @@
 #import <UIKit/UIKit.h>
 
 #import <React/RCTBridgeModule.h>
-#import "WXApi.h"
+#import <WechatOpenSDK/WXApi.h>
 
 // define share type constants
 #define RCTWXShareTypeNews @"news"
diff --git a/node_modules/expo-react-native-wechat-v2/ios/WeChat.mm b/node_modules/expo-react-native-wechat-v2/ios/WeChat.mm
index 6e87085..e9f492b 100644
--- a/node_modules/expo-react-native-wechat-v2/ios/WeChat.mm
+++ b/node_modules/expo-react-native-wechat-v2/ios/WeChat.mm
@@ -1,6 +1,6 @@
 // WeChat.mm
 #import "WeChat.h"
-#import "WXApiObject.h"
+#import <WechatOpenSDK/WXApiObject.h>
 #import <React/RCTEventDispatcher.h>
 #import <React/RCTBridge.h>
 #import <React/RCTLog.h>
